src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -104,6 +104,7 @@ public class C
             }}
           ],
           ""fullMessage"": ""The field 'C.x' is never used"",
+          ""isSuppressedInSource"": false,
           ""properties"": {{
             ""severity"": ""Warning"",
```

Comment:
SARIF provides a 'warning' and an 'error' designation as a property on the result that you should be populating. Note that if this member is missing, the default value is 'warning'. That means that the error messages emitted by Roslyn in your implementation will be bucketed incorrectly (despite you adding a severity property of Error to the property bag)